pages/about.tsx

```diff
@@ -68,98 +69,131 @@ const AboutPage = () => {
-          <RelativeLink to="#60" title="PUSH1" />,{' '}
+          The stack is a list of 32-byte elements used to store smart contract
+          instruction inputs and outputs. The stack is only accessible during
+          smart contract execution. When a new value is put on the stack, it is
+          put on top, and only the top values are used by the instructions. The
+          stack currently has a maximum limit of 1024 values. All instructions
+          interact with the stack, but it can be directly manipulated with
+          instructions like <RelativeLink to="#60" title="PUSH1" />,{' '}
           <RelativeLink to="#50" title="POP" />,{' '}
           <RelativeLink to="#80" title="DUP1" />, or{' '}
           <RelativeLink to="#90" title="SWAP1" />.
         </p>
       </SectionWrapper>
 
-      <SectionWrapper header={<H3>The memory</H3>} anchorKey="memory">
+      <SectionWrapper header={<H3>The Memory</H3>} anchorKey="memory">
         <p className="pb-8">
-          The memory is a region that only exists during the smart contract
-          execution, and is accessed with a byte offset. While all the 32-byte
-          address space is available and initialized to 0, the size is counted
-          with the highest address that was accessed. It is generally read and
-          written with <RelativeLink to="#51" title="MLOAD" /> and{' '}
-          <RelativeLink to="#52" title="MSTORE" /> instructions, but is also
-          used by other instructions like{' '}
+          Unlike EVM code, EVM memory is only accessible during smart contract
+          execution. Memory is not persistent, and ceases at the end of the call
+          context. At the start of a call context, memory is initialized to 0.
+          Reading and Writing from memory is usually done with{' '}
+          <RelativeLink to="#51" title="MLOAD" /> and{' '}
+          <RelativeLink to="#52" title="MSTORE" /> instructions respectively,
+          but can also be accessed by other instructions like{' '}
           <RelativeLink to="#F0" title="CREATE" /> or{' '}
           <RelativeLink to="#F3" title="EXTCODECOPY" />.
         </p>
       </SectionWrapper>
 
-      <SectionWrapper header={<H3>The storage</H3>} anchorKey="storage">
+      <SectionWrapper header={<H3>The Storage</H3>} anchorKey="storage">
         <p className="pb-8">
-          The storage is the persistent memory of the smart contract. It is a
-          map of the 32-byte slot to 32-byte value, and each value written is
-          kept until it is set to 0 or the contract self-destruction. Reading
-          from an unset key also returns 0. It is read and written with the
-          instructions <RelativeLink to="#54" title="SLOAD" /> and{' '}
+          Storage is a map of 32-byte slots to 32-byte values. Storage is the
+          persistent memory of smart contracts: each value written by the
+          contract is retained past the completion of a function call, unless
```

Comment:
```suggestion
          contract is retained past the completion of a call, unless
```
There is no concept of functions in the EVM so far, but it may be added in the future, so let's not mix it up.